### PR TITLE
 Changes need to get build working for nucleo board

### PR DESCRIPTION
**Pull request by chris-hl**

### CODE REVIEW

File: src/env_zephyr.c [hunk 1]
```diff
@@ -9,6 +9,9 @@ bool rcutils_set_env(const char * env_name, const char * env_value)
 
 const char* rcutils_get_env(const char * env_name, const char ** env_value)
 {
+	if (NULL == *env_value) {
```
Comment (yashi) [inline src/env_zephyr.c:12]: Oh, crap.  The function returns `NULL` for success?!?  That was unexpected. If we were not targeting embedded system, we would return "Unsupported" or "Not implemented".  But having strings to indicate errors is a non-sense on system with tight memory.

Thank you for fixing this!

File: zephyr/CMakeLists.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 set(CMAKE_C_STANDARD 11)
 
-option(RCUTILS_NO_THREAD_SUPPORT "Disable thread support." OFF)
+option(RCUTILS_NO_THREAD_SUPPORT "Disable thread support." ON)
```
Comment (yashi) [inline zephyr/CMakeLists.txt:3]: Hmm.  It's odd.  Picolibc supports TLS.  Let me check.

Comment (chris-hl) [reply zephyr/CMakeLists.txt:3]: commit message updated to describe why we disable thread support

**Comment by yashi** [comment]
Thanks!

**Pull request merged**